source/common/network/listener_impl.cc

```diff
@@ -23,20 +23,25 @@ Address::InstanceConstSharedPtr ListenerImpl::getLocalAddress(int fd) {
 void ListenerImpl::listenCallback(evconnlistener*, evutil_socket_t fd, sockaddr* remote_addr,
                                   int remote_addr_len, void* arg) {
   ListenerImpl* listener = static_cast<ListenerImpl*>(arg);
-  ConnectionSocketPtr socket(new AcceptedSocketImpl(
-      fd,
-      // Get the local address from the new socket if the listener is listening on IP ANY
-      // (e.g., 0.0.0.0 for IPv4) (local_address_ is nullptr in this case).
-      !listener->local_address_ ? listener->getLocalAddress(fd) : listener->local_address_,
-      // The accept() call that filled in remote_addr doesn't fill in more than the sa_family field
-      // for Unix domain sockets; apparently there isn't a mechanism in the kernel to get the
-      // sockaddr_un associated with the client socket when starting from the server socket.
-      // We work around this by using our own name for the socket in this case.
+  // Get the local address from the new socket if the listener is listening on IP ANY
+  // (e.g., 0.0.0.0 for IPv4) (local_address_ is nullptr in this case).
+  const Address::InstanceConstSharedPtr& local_address =
+      !listener->local_address_ ? listener->getLocalAddress(fd) : listener->local_address_;
```

Comment:
Tiny nit: prefer to negate the conditional here, easier to parse.

Reply:
Done.